### DISCUSSION TITLE
Does Java has a semicolon(;) at the end of code line?

### DISCUSSION THREAD

**Discussion by HaiderAli785** [Schweinepriester/github-profile-achievements#272] [Q&A] [answered]
Does Java has a semicolon(;) at the end of code line?

**Comment by haideralicodes** [comment, marked as answer]
Yes, java has a semicolon at the end of line of code which shows that the statement has been terminated. And java executes the code line by line.

**Comment by haideralicodes** [comment]
Semicolon is a punctuation mark (;) indicating a pause, typically between two main clauses, that is more pronounced than that indicated by a comma. In programming, Semicolon symbol plays a vital role. It is used to show the termination of instruction in various programming languages as well, like C, C++, Java, JavaScript and Python.

**Reply by thenithinbalaji** [reply]
I wish I had 2 accounts like you
